source/server-selection/tests/in_window/one-least-two-tied.json

```diff
@@ -0,0 +1,42 @@
+{
+  "description": "Least operations gets most selections, two tied share the rest",
```

Comment:
Note that even with 10000 iterations I got this false positive [in this evergreen patch](https://evergreen.mongodb.com/version/5fadb207c9ec440ad07f3cb1):
```
 [2020/11/12 22:16:08.832] FAIL [0.230s]: test_in_window_one_least_two_tied_Least_operations_gets_most_selections,_two_tied_share_the_rest (test_server_selection_in_window.TestAllScenarios)
 [2020/11/12 22:16:08.832] ----------------------------------------------------------------------
 [2020/11/12 22:16:08.832] Traceback (most recent call last):
 [2020/11/12 22:16:08.832]   File "/data/mci/491f6078eca3c0414292a7732afbf55b/src/test/__init__.py", line 444, in wrap
 [2020/11/12 22:16:08.832]     return f(*args, **kwargs)
 [2020/11/12 22:16:08.832]   File "/data/mci/491f6078eca3c0414292a7732afbf55b/src/test/test_server_selection_in_window.py", line 92, in run_scenario
 [2020/11/12 22:16:08.832]     self.run_scenario(scenario_def)
 [2020/11/12 22:16:08.832]   File "/data/mci/491f6078eca3c0414292a7732afbf55b/src/test/test_server_selection_in_window.py", line 87, in run_scenario
 [2020/11/12 22:16:08.832]     self.assertAlmostEqual(actual_freq, freq, delta=0.02)
 [2020/11/12 22:16:08.832] AssertionError: 0.6805 != 0.66 within 0.02 delta (0.020499999999999963 difference)
```

Reply:
done